specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -1233,18 +1233,18 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/linkedWorkspaces": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/linkedServices": {
       "get": {
```

Comment:
Seems this API is designed to be paginated (inferred from the return type name `PaginatedLinkedServiceList`). If so, could we add a `x-ms-pagable` extension here?

Reply:
Also, if this is not pagable, could we change the type name to reduce confusion?

Reply:
@ArcturusZhang We don't support pagination yet, but I was told that if we don't support pagination in list API, some API checks will fail. Could you please confirm that? If that's not the case, we'd like to keep the previous simple array[] return type rather than a PaginatedList.